src/Tasks/Resources/Strings.resx

```diff
@@ -1681,6 +1681,10 @@
     <value>The AssemblyFolder config file ('{0}') specified in Microsoft.Common.CurrentVersion.targets was invalid. The error was: {1}</value>
     <comment></comment>
   </data>
+  <data name="ResolveAssemblyReference.AssemblyDoesNotContainPEMetadata">
+    <value>Assembly file '{0}' could not be opened -- PE image doesn't contain managed metadata.</value>
```

Comment:
Is this identical to the Framework error?

Reply:
No, not exactly. This was copied from the Roslyn error. The framework error was not as good. But it's just tacked onto the end of an existing error so it's never in isolation. And when I read the two (Roslyn and Framework) this was the cleaner of the two.

Reply:
Works for me.